db/schema.rb

```diff
@@ -0,0 +1,43 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_08_03_022448) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  # Custom types defined in this database.
+  # Note that some types may not work with other database engines. Be careful if changing database.
+  create_enum "keyword_status", ["processing", "succeeded", "failed", "pending"]
+  create_enum "search_entry_kind", ["ads", "non_ads"]
+  create_enum "search_entry_position", ["top", "bottom", "main_search"]
+
+  create_table "keyword_search_entries", id: :serial, force: :cascade do |t|
+    t.enum "kind", null: false, enum_type: "search_entry_kind"
+    t.string "urls", default: [], array: true
+    t.enum "position", enum_type: "search_entry_position"
```

Comment:
Small thing, should we add `null: false`?

Reply:
Fixed in 7a92f96ad741fbd954383290d8ccfde4d7326772

Also merge migration files to keep it tidy